examples/pip_parse/pip_parse_test.py

```diff
@@ -19,20 +19,28 @@
 import unittest
 from pathlib import Path
 
-from rules_python.python.runfiles import runfiles
+from python.runfiles import runfiles
 
 
 class PipInstallTest(unittest.TestCase):
     maxDiff = None
 
+    def _remove_leading_dirs(self, paths):
+        # Removes the first two directories (external/<reponame>)
+        # to normalize what workspace and bzlmod produce.
+        #return [str(Path(*Path(v).parts[2:])) for v in paths]
+        return [
+            '/'.join(v.split('/')[2:])
```

Comment:
FYI, I just wanted to comment that we should use an os specific path separator, but realized that the values come from `rlocationpaths` via an env var which will always use `/` separators, right?

Reply:
Correct. Originally I used pathlib.Path to do this, but on windows, it changed things to `\` and then the paths didn't match the expected values.